### ISSUE TITLE
I think I'm going to keep working on this project.

### ISSUE THREAD

**Issue by ghost** [ephemerant/C346-Roguelike#23]
I kind of like it, and I think I'll keep playing around with it after this semester. Anyone else?

**Comment by ephemerant** [comment]
Yeah, I'm in. I'll be booked through most of all next week, but I should have some time after that.

**Comment by ghost** [comment]
Cool, maybe Stephen will want too as well.

**Comment by ephemerant** [comment]
Never count Jason out, either!

**Comment by ghost** [comment]
Yeah!

**Comment by jimbobmaggot** [comment]
Yup, still here. I'll be focusing on the final and a few other homework stuff for the next week or so, then I'll try to get back to this

**Comment by chrissexton** [comment]
You guys might want to check out this local game development group. They seem to be just getting off the ground as a networking and learning group. 

http://louisvillemakesgames.org
